mathext/internal/amos/amoslib/fortran.go

```diff
@@ -3,7 +3,7 @@
 // license that can be found in the LICENSE file.
 
 // +build fortran
-// TODO(jonlawlor): remove fortran build tag when gonum only supports go 1.7+.
+// TODO(jonlawlor): remove fortran build tag when Gonum only supports go 1.7+.
```

Comment:
Leaving the build tag here since we don't ask travis to install gfortran. If we want to do this TODO we will need to do that. I'm not convinced it's worthwhile.